spec/rails_helper.rb

```diff
@@ -90,6 +92,7 @@
   config.include Warden::Test::Helpers, type: :request
   config.include Warden::Test::Helpers, type: :feature
   config.include Capybara::DSL
+  config.include CarrierWave::Test::Matchers
 
```

Comment:
does this help with `subject.format`? i see this can be used for `be_format`, but I didn't see this usage.

Reply:
I'm manually handling format, so I'm not sure this would help. Maybe though, where's the docs?